### PR TITLE
Fix compiling the unit tests with C89 compilers

### PR DESCRIPTION
**Pull request by ccawley2011**
Split out from PR #430.

### CODE REVIEW

**Comment by sezero** [comment]
This one looks sensible to me. @AliceLR?

File: test-dev/test.h [hunk 1]
```diff
@@ -34,6 +35,19 @@ static inline int is_big_endian() {
 	return (*(char *)&w == 0x00);
 }
 
+static inline double libxmp_round(double val)
+{
+	return (val >= 0.0)? floor(val + 0.5) : ceil(val - 0.5);
+}
+
+/* Get period from note */
+static int note_to_period(int n)
```
Comment (sezero) [inline test-dev/test.h:44]: Should this be marked `inline`?

**Review by sezero — APPROVED** [review]
This looks good to me.

**Review by AliceLR — APPROVED** [review]

**Comment by AliceLR** [comment]
This looks OK to me.

**Pull request merged**